client/netmap.go

```diff
@@ -196,3 +196,83 @@ func (c *Client) NetworkInfo(ctx context.Context, prm PrmNetworkInfo) (*ResNetwo
 
 	return &res, nil
 }
+
+// PrmNetMapSnapshot groups parameters of NetMapSnapshot operation.
+type PrmNetMapSnapshot struct {
+	prmCommonMeta
+}
+
+// ResNetMapSnapshot groups resulting values of NetMapSnapshot operation.
+type ResNetMapSnapshot struct {
+	statusRes
+
+	netMap netmap.NetMap
+}
+
+// NetMap returns current server's local network map.
+func (x ResNetMapSnapshot) NetMap() netmap.NetMap {
+	return x.netMap
+}
+
+// NetMapSnapshot requests current network view of the remote server.
+//
+// Any client's internal or transport errors are returned as `error`.
+// If PrmInit.ResolveNeoFSFailures has been called, unsuccessful
+// NeoFS status codes are returned as `error`, otherwise, are included
+// in the returned result structure.
+//
+// Context is required and MUST NOT be nil. It is used for network communication.
+//
+// Exactly one return value is non-nil. Server status return is returned in ResNetMapSnapshot.
+// Reflects all internal errors in second return value (transport problems, response processing, etc.).
+//
+// Return statuses:
+//   - global (see Client docs).
+func (c *Client) NetMapSnapshot(ctx context.Context, prm PrmNetMapSnapshot) (*ResNetMapSnapshot, error) {
```

Comment:
IMO `NetMap` stands for network map, so camel-case is pretty expected here.

Reply:
well, i always thought that "Netmap" is the right (and the only one) choice: like the "internet" (interconnected networks), "midnight" (middle night), etc. so i expect "Netmap", "NetworkMap" but not "NetMap" since i doubt we can write "... our net map is..."; we write "netmap" or "network map"

@EliChin, please, help us here

Reply:
I got your point. I like `NetworkMap`, too, but pls don't compare with internet - it's generally accepted name of the exact network, and midnight - I don't even know when it'd been appeared in English :smile:

We'll resolve it separately anyway since type is called `NetMap`.